nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/enrichers/BlobAppLinksJsonEnricher.java

```diff
@@ -32,21 +38,17 @@
 import org.nuxeo.ecm.core.io.registry.reflect.Setup;
 import org.nuxeo.runtime.api.Framework;
 
-import java.io.IOException;
-import java.util.List;
-
-import static org.nuxeo.ecm.core.io.registry.reflect.Instantiations.SINGLETON;
-import static org.nuxeo.ecm.core.io.registry.reflect.Priorities.REFERENCE;
+import com.fasterxml.jackson.core.JsonGenerator;
 
 /**
  * Enrich {@link Blob} json with list of {@link AppLink}.
- *
+ * <p>
  * Enabled when parameter enrichers-blob=appLinks is present.
```

Comment:
Not related directly to your PR but, don't we use `enrichers.blob=appLinks`?

Reply:
Agree, yet in most of the enricher's Javadoc this syntax is used...

Reply:
Not really, no, see https://jira.nuxeo.com/browse/NXP-18425

Reply:
Right!